projects/igniteui-angular/src/lib/query-builder/query-builder-tree.component.ts

```diff
@@ -424,7 +424,7 @@ export class IgxQueryBuilderTreeComponent implements AfterViewInit, OnDestroy {
     /**
      * @hidden @internal
      */
-    public searchValue: any;
+    public searchValue: { value: any } = { value: null };
```

Comment:
Why do we need this? If we need null, why not just:
```
public searchValue: any = null;
``` 
Or is it in case we eventually need to add more props to the search value object?

Reply:
We need it to be an object so we can pass it between components as a reference and keep it's data bound both ways